src/main.rs

```diff
@@ -0,0 +1,37 @@
+//I'd like the most pedantic warning level
+#![warn(clippy::pedantic, clippy::needless_borrow)]
+// But I don't care about these ones for now (most applicable since the code isn't fleshed out)
+#![allow(clippy::module_name_repetitions)]
+#![allow(clippy::needless_pass_by_value)]
+
+#[macro_use]
+extern crate failure;
+#[macro_use]
+extern crate log;
+
+extern crate serde;
```

Comment:
If you aren't using macros, you don't need this line

Reply:
Line 9? I do need the `#[macro_use]` otherwise it doesn't compile

Reply:
Line 20

Reply:
Line 20